Assets/PatchKit/UnityEditorExtension/Editor/UI/CreateAppScreen.cs

```diff
@@ -70,7 +104,7 @@ public override void Draw()
                 {
                     GUILayout.FlexibleSpace();
 
-                    using (Style.Colorify(Color.green))
+                    using (Style.Colorify(new Color(0.502f, 0.839f, 0.839f)))

```

Comment:
Put this color in some static readonly variable in `Style` class.

Reply:
ok